src/BurnableMintableCappedERC20.sol

```diff
@@ -48,14 +40,7 @@ contract BurnableMintableCappedERC20 is ERC20, Ownable {
             );
     }
 
-    function mint(address account, uint256 amount) public onlyOwner {
-        uint256 capacity = cap;
-        require(capacity == 0 || totalSupply + amount <= capacity, 'CAP_EXCEEDED');
-
-        _mint(account, amount);
-    }
-
-    function burn(bytes32 salt) public onlyOwner {
+    function burn(bytes32 salt) public {
```

Comment:
Anyone can burn a Burner's token, which can mess up the validators' awareness of it happening when they try.

I think checking `msg.sender` against `depositAddress(salt)` is key.

Reply:
There was a `onlyOwner` there which I accidentally removed.. I'll put it back.